### ISSUE TITLE
[Time field, Date field] Focus indicator has insufficient contrast on spinbuttons

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3601]
### 😯 Current Behavior

When user place keyboard focus on hour or minute spinbuttons focus indicator in form of light blue background appears. This light blue has contrast of 1,5:1 againt white background while the minimum contrast for that should be 3:1. 

It's important to have visible focus indicator for each spinbutton not only for the whole Time Field and Date Field as user may be unaware that there are two spinbuttons.

<img width="384" height="243" alt="Image" src="https://github.com/user-attachments/assets/9f9b07b1-edf8-41d3-8294-0d1f2f8b390b" />

<img width="172" height="84" alt="Image" src="https://github.com/user-attachments/assets/34d6c0f6-aae1-410a-b1c0-869b7f90a1b3" />

### 🤔 Expected Behavior/possible solution?

It's recommended to change the color of blue background to darker (e.g. #3C96) which works with white background and black numbers at the same time. 

Alternatively focus ring can be used instead of background color.

### 🖥️ Steps to Reproduce

1. Go to https://storybook.sanomalearning.design/?path=/story/form-time-field--basic
2. Set focus on hour or minute spinbutton
3. Verify focus indicator contrast ratio against background

### Were you able to reproduce this in our storybook?

Yes

### If yes, provide the storybook URL

https://storybook.sanomalearning.design/?path=/story/form-time-field--basic

### ⚖️ Impact

Not a huge problem, we have a workaround

### Framework

Other

### What browsers are you seeing the problem on?

All

### Component version

0.1.3

### 👤 Your name

Kacper

### 🧢 Your product/team

SLDS

### 🎨 Your Theme(s)

Sanoma Learning
